BFS_Connected_Components/Q1_number_of_transformations_bfs.py

```diff
@@ -0,0 +1,37 @@
+import collections
+
+# BFS
+def get_min_operations_bfs(start_num, target_num, additive_nums, multiplicative_nums):
```

Comment:
Function names should be declarative, not imperative. From Google: "Declarative programming is a paradigm describing WHAT the program does, without explicitly specifying its control flow. Imperative programming is a paradigm describing HOW the program should do something by explicitly specifying each instruction (or statement) step by step, which mutate the program's state."

Basically, function names should state WHAT the function does, but not HOW it does it. So, this function should not include `bfs` in its name.